api/python/provisioner/commands/__init__.py

```diff
@@ -1163,6 +1163,30 @@ def _passwordless_ssh():
             _generate_ssh_config()
             _copy_minion_nodes()
 
+        # Update PATH just as a precaution
+        StateFunExecuter.execute(
+            'file.blockreplace',
+            fun_kwargs=dict(
+                name='~/.bashrc',
+                marker_start='# DO NOT EDIT: Start',
+                marker_end='# DO NOT EDIT: End',
+                content='export PATH=$PATH:/usr/local/bin',
```

Comment:
@ipoddubnyy @divya-vijay I think we shold mostly rely on `/usr/bin` in production. So makes sense to prepend (not append) both `/usr/bin` should be the first

Reply:
@andkononykhin yeah, but /usr/bin is, by default, present in all PATH variables. so I did not explicitly mention it. but in cases where /usr/local/bin is also present, precedence goes to local/. 
you suggest to add it this way -->  "/usr/bin:/usr/local/bin:$PATH " ?